recipes/adoptopenjdk.rb

```diff
@@ -1,46 +0,0 @@
-# Cookbook:: java
-# Recipe:: adoptopenjdk
-
-include_recipe 'java::notify'
```

Comment:
To make this migration less painful you really want to at least keep this recipe with a proper raise that includes messaging on what has changed and where a user can go to learn about the new cookbook. A recipe missing error won't help them to get onto the new resources since they generally won't understand that they're even pulling in a new version of the java cookbook.

Reply:
Ace, I've put out a Log.fatal message, with a raise so we have both a nice friendly Chef log message above the halting to the recipe execution.